pkgs/build-support/fetchgit/default.nix

```diff
@@ -103,7 +104,7 @@ stdenvNoCC.mkDerivation {
 
   inherit preferLocalBuild meta allowedRequisites;
 
-  passthru = {
+  passthru = passthru // {
```

Comment:
Who wins in the case of setting gitRepoUrl?

Reply:
I don't think it's even reasonable to `passthru` `gitRepoUrl`. Nevertheless, let's give users the choice to do so.